.travis.yml

```diff
@@ -9,5 +9,13 @@ install:
 script: bash -ex .travis-docker.sh
 env:
   global:
-    - PACKAGE="xapi-stdext"
-    - PINS="stdext:. xapi-stdext:. xapi-stdext-date:. xapi-stdext-encodings:. xapi-stdext-pervasives:. xapi-stdext-std:. xapi-stdext-threads:. xapi-stdext-unix:. xapi-stdext-zerocheck:."
+    - PINS="stdext:. xapi-stdext-date:. xapi-stdext-encodings:. xapi-stdext-pervasives:. xapi-stdext-std:. xapi-stdext-threads:. xapi-stdext-unix:. xapi-stdext-zerocheck:."
+  jobs:
```

Comment:
Does each of these kick off new Travis builds? That would be quite resource intensive

Reply:
I guess we could get away by having only 2 jobs, one for stdext that compiles everything and one for encodings that runs its tests.

I don't think the CI scripts allow to install and test more than a package at a time, this is why xen-api is using a custom script that runs `make` and `make test`.

Reply:
Testing a reduced travis configuration at https://travis-ci.com/github/psafont/stdext/builds/176528274

Reply:
yep, it ci-opam only [supports](https://github.com/ocaml/ocaml-ci-scripts/blob/master/src/ci_opam.ml#L38) building one package:
```
Fatal error: exception Failure("No opam file found for stdext xapi-stdext-date xapi-stdext-encodings xapi-stdext-pervasives xapi-stdext-std xapi-stdext-threads xapi-stdext-unix xapi-stdext-zerocheck, aborting.")
```
I'll go for the 2-jobs option